scrapy/commands/genspider.py

```diff
@@ -61,6 +62,13 @@ def run(self, args, opts):
             raise UsageError()
 
         name, domain = args[0:2]
+
+        parsed_url = urlparse(domain)
+
+        # Automatically it is prepended 'http' if the scheme is missing
```

Comment:
I think this comment is unnecessary, this particular action is done in one line and it should be self-explanatory.